### PR TITLE
[FMT] apply black to plotting - part 1

### PR DESCRIPTION
**Pull request by Remi-Gau**
Relates to https://github.com/nilearn/nilearn/issues/2528

Changes proposed in this pull request:

- update configs to start including nilearn/plotting
- apply black

### CODE REVIEW

File: nilearn/plotting/html_document.py [hunk 1]
```diff
@@ -105,9 +110,11 @@ def get_iframe(self, width=None, height=None):
         if height is None:
             height = self.height
         escaped = escape(self.html, quote=True)
-        wrapped = (f'<iframe srcdoc="{escaped}'
-                   f'width="{width}" height="{height}" '
-                   'frameBorder="0"></iframe>')
+        wrapped = (
+            f'<iframe srcdoc="{escaped}'
```
Comment (ymzayek) [inline nilearn/plotting/html_document.py:114]: i think that should be:
```suggestion
            f'<iframe srcdoc="{escaped}" '
```

File: nilearn/plotting/tests/test_html_connectome.py [hunk 2]
```diff
@@ -45,65 +48,94 @@ def _make_connectome():
...
+            0,
+            40,
+            20,
+            0,
+            40,
+            40,
+            0,
+        ],
+        dtype="<f4",
+    )
```
Comment (bthirion) [inline nilearn/plotting/tests/test_html_connectome.py:89]: :-(

File: nilearn/plotting/cm.py [hunk 3]
```diff
@@ -161,48 +168,86 @@ def _revcmap(data):
...
+    "#33334c",
+    "#4c4c7f",
+    "#7f7fcc",
+    "#00ff00",
+    "#10b010",
+    "#ffff00",
+    "#ff9900",
+    "#ff6900",
+    "#ff0000",
+]
```
Comment (bthirion) [inline nilearn/plotting/cm.py:243]: :-(

**Review by bthirion — APPROVED** [review]
LGTM, thx.

File: nilearn/plotting/cm.py (hunk 3 above)
Comment (Remi-Gau) [reply nilearn/plotting/cm.py:243]: yeah I know: I noticed that too but not sure there is nothing more to than bite the bullet for now.

File: nilearn/plotting/tests/test_html_connectome.py (hunk 2 above)
Comment (Remi-Gau) [reply nilearn/plotting/tests/test_html_connectome.py:89]: I agree

**Pull request merged**